### PR TITLE
Update 1.21

### PR DESCRIPTION
**Pull request by MaxxQc**

### CODE REVIEW

File: src/main/java/fr/skytasul/glowingentities/GlowingEntities.java [hunk 1]
```diff
@@ -1014,7 +1014,24 @@ private enum ProtocolMappings {
...
 					"packedItems"
-					)
+					),
+			V1_21(
+					21,
+					0,
+					false,
+					"ap",
+					"aq",
+					"ar",
```
Comment (SkytAsul) [inline src/main/java/fr/skytasul/glowingentities/GlowingEntities.java:1024]: No need to write all fields. The only that has changed is "ar".

File: src/main/java/fr/skytasul/glowingentities/GlowingEntities.java [hunk 2]
```diff
@@ -1014,7 +1014,24 @@ private enum ProtocolMappings {
...
+					"c",
+					"e",
+					"b",
+					"n",
+					"a",
+					"a",
+					"c",
+					"d"
+					),
 			// remapping not complete: should also use remapped class names
```
Comment (SkytAsul) [inline src/main/java/fr/skytasul/glowingentities/GlowingEntities.java:1035]: Please move this comment at line 1018.

**Review by SkytAsul — CHANGES_REQUESTED** [review]
You should also change pom.xml paper-api dependency to 1.21.

**Review by SkytAsul — APPROVED** [review]

**Comment by SkytAsul** [comment]
Have you tried it? Both GlowingEntities and GlowingBlocks?

**Comment by MaxxQc** [comment]
> Have you tried it? Both GlowingEntities and GlowingBlocks?

I tried GlowingEntities, but I did not try GlowingBlocks since I don't have a Paper setup.
I will update when I do.

**Comment by MaxxQc** [comment]
image

Works great on paper 1.21

**Comment by SkytAsul** [comment]
Awesome! Thank you very much for this PR :)

**Pull request merged**